src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/FixAllContext.cs

```diff
@@ -88,8 +96,45 @@ public partial class FixAllContext
             IEnumerable<string> diagnosticIds,
             DiagnosticProvider fixAllDiagnosticProvider,
             CancellationToken cancellationToken)
+            : this(document, diagnosticSpan: null, codeFixProvider, scope,
```

Comment:
can we add this to teh *roslyn* banned API list?  taht way we make sure that all our code calls teh new constructor?  i get the benefit of having this still in teh public API.  but we want to be careful we're not accidentally using this.

Reply:
Sure, I'll do that.

Reply:
Turns out both the existing public constructors for FixAllContext are already part of the banned API list, recommendation is to use the internal constructor overload:
https://github.com/dotnet/roslyn/blob/ac603f8bdde93a35e2e4b9ef95a4674b561ab433/eng/config/BannedSymbols.txt#L9-L10

I'll add even the new public API to this file.